### PR TITLE
develop: update tidb cloud import steps

### PR DESCRIPTION
**Pull request by Mini256**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [x] v6.4 (TiDB 6.4 versions)
- [x] v6.3 (TiDB 6.3 versions)
- [x] v6.2 (TiDB 6.2 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s): https://github.com/pingcap/docs-cn/pull/11754

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by Icemap — APPROVED** [review]
LGTM.
And don't forget about the Chinese edition.

File: develop/dev-guide-bookshop-schema-design.md [hunk 1]
```diff
@@ -94,9 +94,10 @@ You can delete the original table structure through the `--drop-tables` paramete
 
 ### Method 2: Via TiDB Cloud Import
 
-On the database details page of TiDB Cloud, click the **Import** button to enter the **Data Import Task** page. On this page, perform the following steps to import the Bookshop sample data from AWS S3 to TiDB Cloud.
+On the cluster detail page of TiDB Cloud, switch to the **Import** tab, and click the **Import Data** button to enter the **Data Import** page. On this page, perform the following steps to import the Bookshop sample data from AWS S3 to TiDB Cloud. (Please refer to the [Migrate From Amazon S3 or GCS](https://docs.pingcap.com/tidbcloud/migrate-from-amazon-s3-or-gcs#step-3-import-data-into-tidb-cloud) guide for detailed steps):
```
Comment (qiancai) [inline develop/dev-guide-bookshop-schema-design.md:97]:
```suggestion
On the cluster detail page of TiDB Cloud, click **Import Data** in the **Import** area to enter the **Data Import** page. On this page, perform the following steps to import the Bookshop sample data from AWS S3 to TiDB Cloud.
```

File: develop/dev-guide-bookshop-schema-design.md [hunk 2]
```diff
@@ -122,20 +131,9 @@ On the database details page of TiDB Cloud, click the **Import** button to enter
...
-
-    ![Bookshop data importing](/media/develop/importing_bookshop_data.png)
-
-    If the following error message appears during the import process, run the `DROP DATABASE bookshop;` command to clear the previously created sample database and then import data again.
+    ![The preview step of import data](/media/develop/import-data-preview.png)
 
-    > table(s) [`bookshop`.`authors`, `bookshop`.`book_authors`, `bookshop`.`books`, `bookshop`.`orders`, `bookshop`.`ratings`, `bookshop`.`users`] are not empty.
+5. Click the **Start Import** button to start the import process and wait for TiDB Cloud to complete the importing.
 
 For more information about TiDB Cloud, see [TiDB Cloud Documentation](https://docs.pingcap.com/tidbcloud).
```
Comment (qiancai) [inline develop/dev-guide-bookshop-schema-design.md:138]:
```suggestion
For more information about how to import or migrate data to TiDB Cloud, see [TiDB Cloud Migration Overview](https://docs.pingcap.com/tidbcloud/tidb-cloud-migration-overview).
```

File: develop/dev-guide-bookshop-schema-design.md (hunk 1 above)
Comment (qiancai) [reply develop/dev-guide-bookshop-schema-design.md:97]: please check if we can guide users to see more information in L140.

File: develop/dev-guide-bookshop-schema-design.md [hunk 3]
```diff
@@ -114,6 +115,14 @@ On the database details page of TiDB Cloud, click the **Import** button to enter
     arn:aws:iam::494090988690:role/s3-tidb-cloud-developer-access
     ```
 
+    ![The fill from step of import data](/media/develop/import-data-fill-from.png)
```
Comment (qiancai) [inline develop/dev-guide-bookshop-schema-design.md:118]: The UI is supposed to be changed today for a feature, which makes the screenshot outdated. Could we not provide the screenshot?

File: develop/dev-guide-bookshop-schema-design.md [hunk 4]
```diff
@@ -114,6 +115,14 @@ On the database details page of TiDB Cloud, click the **Import** button to enter
...
+
+3. Click the **Next** button switch to the **File and filter** step to confirm the information of the file that will be imported.
```
Comment (qiancai) [inline develop/dev-guide-bookshop-schema-design.md:120]:
```suggestion
3. Click **Next** to go to the **File and filter** step to confirm the information of the files to be imported.
```

File: develop/dev-guide-bookshop-schema-design.md [hunk 5]
```diff
@@ -114,6 +115,14 @@ On the database details page of TiDB Cloud, click the **Import** button to enter
...
+
+    ![The file and filter step of import data](/media/develop/import-data-file-and-filter.png)
+
+4. Click the **Next** button again switch to the **Preview** step to confirm the sample data that will be imported is fine.
```
Comment (qiancai) [inline develop/dev-guide-bookshop-schema-design.md:124]:
```suggestion
4. Click **Next**  again to go to the **Preview** step to confirm the preview of the data to be imported.
```

File: develop/dev-guide-bookshop-schema-design.md [hunk 6]
```diff
@@ -94,9 +94,10 @@ You can delete the original table structure through the `--drop-tables` paramete
 
 ### Method 2: Via TiDB Cloud Import
 
-On the database details page of TiDB Cloud, click the **Import** button to enter the **Data Import Task** page. On this page, perform the following steps to import the Bookshop sample data from AWS S3 to TiDB Cloud.
+On the cluster detail page of TiDB Cloud, click **Import Data** in the **Import** area to enter the **Data Import** page. On this page, perform the following steps to import the Bookshop sample data from AWS S3 to TiDB Cloud (Please refer to the [Migrate From Amazon S3 or GCS](https://docs.pingcap.com/tidbcloud/migrate-from-amazon-s3-or-gcs#step-3-import-data-into-tidb-cloud) guide for detailed steps).
```
Comment (qiancai) [inline develop/dev-guide-bookshop-schema-design.md:97]:
```suggestion
On the cluster detail page of TiDB Cloud, click **Import Data** in the **Import** area to enter the **Data Import** page. On this page, perform the following steps to import the Bookshop sample data from AWS S3 to TiDB Cloud.
```

**Comment by Mini256** [comment]
@qiancai, thanks for your review!

File: develop/dev-guide-bookshop-schema-design.md [hunk 7]
```diff
@@ -122,22 +127,9 @@ On the database details page of TiDB Cloud, click the **Import** button to enter
...
-4. Enter database login information.
-5. Click the **Import** button to confirm the import.
-6. Wait for TiDB Cloud to complete the import.
-
-    ![Bookshop data importing](/media/develop/importing_bookshop_data.png)
-
-    If the following error message appears during the import process, run the `DROP DATABASE bookshop;` command to clear the previously created sample database and then import data again.
-
-    > table(s) [`bookshop`.`authors`, `bookshop`.`book_authors`, `bookshop`.`books`, `bookshop`.`orders`, `bookshop`.`ratings`, `bookshop`.`users`] are not empty.
+5. Click the **Start Import** button to start the import process and wait for TiDB Cloud to complete the importing.
```
Comment (qiancai) [inline develop/dev-guide-bookshop-schema-design.md:130]:
```suggestion
5. Click **Start Import** to start the import process and wait for TiDB Cloud to complete the importing.
```

**Review by qiancai — APPROVED** [review]

**Review by Oreoxmt — APPROVED** [review]
LGTM

**Pull request merged**